examples/borderless.rs

```diff
@@ -13,7 +13,7 @@ fn app(cx: Scope) -> Element {
         link { href:"https://unpkg.com/tailwindcss@^2/dist/tailwind.min.css", rel:"stylesheet" }
         header {
             class: "text-gray-400 bg-gray-900 body-font",
-            onmousedown: move |_| window.drag(),
+            onmousedown: move |_| window.drag_window(),
```

Comment:
Keeping "drag" is fine. Right now you have a stutter "window dot drag window". 

Ditto for close.

`set_minimized` is probably a fine switch.